infrastructure/zk/src/run/run.ts

```diff
@@ -12,14 +12,18 @@ export { verifyKeys, dataRestore };
 
 export async function deployERC20(command: 'dev' | 'new', name?: string, symbol?: string, decimals?: string) {
     if (command == 'dev') {
-        await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '
-            [
-                { "name": "DAI",  "symbol": "DAI",  "decimals": 18 },
-                { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
-                { "name": "BAT",  "symbol": "BAT",  "decimals": 18 },
-                { "name": "GNT",  "symbol": "GNT",  "decimals": 18 },
-                { "name": "MLTT", "symbol": "MLTT", "decimals": 18 }
-            ]' > ./etc/tokens/localhost.json`);
+        await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '[
```

Comment:
I'm pretty sure that this change breaks some tests. Have you tried to run them?

Reply:
I put them back now.

Reply:
erm.. yeap, why?

Reply:
@jurajpiar The order of the tokens may affect the test results. Have you run the tests? 😅

Reply:
that suggests fragile tests more than anything. no I did not lol. I will tho

Reply:
the tests are passing just fine it appears